examples/pyplots/sigmoid_function.py

```diff
@@ -0,0 +1,35 @@
+"""
+============
+Infinite horizontal and vertical lines
+============
+A script that showes infinite horizontal and vertical lines together with a sigmoid function.
```

Comment:
I'd focus more on the functions, and their effect/purpose, e.g.
~~~
`~.axes.Axes.axvline` and `~.axes.Axes.axhline` draw infinite vertical /
horizontal lines, at given *x* / *y* positions. They are usually used to mark
special data values, e.g. in this example the center and limit values of the
sigmoid function.
~~~

Reply:
yep, I see.